README.md

```diff
@@ -3,34 +3,24 @@
 ## Usage
 
 ```
-mkdir ~/.s3cfg
-
-docker run -it --rm         \
-  --volume ~/.s3cfg:/.s3cfg \
-  --user $(id -u)           \
-  frew/s3cmd                \
-  configure
-
-docker run -it --rm         \
-  --volume ~/.s3cfg:/.s3cfg \
-  --volume $(pwd):/pwd      \
-  --user $(id -u)           \
-  frew/s3cmd                \
-  sync /pwd/ s3://mah-bukkit
+#setup
+docker run -it --rm -v $(pwd):/root frew/s3cmd  --configure
+
+#syncing
+docker run -it --rm -v $(pwd):/root -v $(pwd)/public:/public frew/s3cmd  sync public/ s3://mah-bukkit
 ```
 
 ## Description
 
-The packaged versions of [s3cmd](http://s3tools.org/) tend to be a little out of
```

Comment:
Why'd you remove my newlines?! ;)

Reply:
;) Good question... probably laziness so it's faster to copy/paste but you're right, it's more legible with them in! :)